src/CoverageTransformer.js

```diff
@@ -13,6 +13,7 @@ export default class CoverageTransformer {
 	constructor(options) {
 		this.basePath = options.basePath;
 		this.warn = options.warn || console.warn;
+		this.warnMissingSourceMaps = options.warnMissingSourceMaps || true;
```

Comment:
@chrismwendt @dylans How can `this.warnMissingSourceMaps` ever be `false`? Any falsy value for `options.warnMissingSourceMaps` will be disregarded in favor of the `true`.

Should it not be 
```
this.warnMissingSourceMaps = (options.warnMissingSourceMaps !== undefined) ? options.warnMissingSourceMaps : true;
```

Reply:
Yes, clearly I was far too permissive with reviewing this PR. Please raise a PR and I'll review and land it, thanks.

Reply:
Raised at #165. Thanks!